rotkehlchen/db/upgrades/v41_v42.py

```diff
@@ -118,6 +118,24 @@ def _add_calendar_table(write_cursor: 'DBCursor') -> None:
     );""")  # noqa: E501
 
 
+@enter_exit_debug_log()
+def _remove_manualcurrent_oracle(write_cursor: 'DBCursor') -> None:
+    write_cursor.execute('SELECT value FROM settings WHERE name="current_price_oracles"')
+    if (data := write_cursor.fetchone()) is None:
+        return  # oracles not configured
+    try:
+        oracles: list[str] = json.loads(data[0])
+    except json.JSONDecodeError as e:
+        log.debug(f'Failed to read oracles from user db. {e!s}')
```

Comment:
should be logged as an error
```suggestion
        log.error(f'Failed to read oracles from user db. {e!s}')
```